packages/profile/src/index.ts

```diff
@@ -14,3 +14,5 @@ export {
   verifyProfileToken,
   extractProfile,
 } from './profileTokens';
+
+export { PublicProfile, PublicPersonProfile } from './types';
```

Comment:
Curious, do we need "Public" in the name? Are there Private alternatives to need a distinction?

Reply:
We should use a name that references objects like the payload in https://gaia.blockstack.org/hub/1Maw8BjWgj6MWrBCfupqQuWANthMhefb2v/0/profile.json

It is public data, users should be aware of that. Furthermore, there is a `Profile` class as a helper class for the `PublicProfile` type.

There is also #1382 that is related to this topic.

Open to any suggestions.